docs/api-reference/next/future/image.md

```diff
@@ -39,13 +41,134 @@ Compared to `next/image`, the new `next/future/image` component has the followin
 - Removes `loader` config in favor of [`loader`](#loader) prop
 - Note: the [`onError`](#onerror) prop might behave differently
 
-The default layout for `next/image` was `intrinsic`, which would shrink the `width` if the image was larger than it's container. Since no styles are automatically applied to `next/future/image`, you'll need to add the following CSS to achieve the same behavior:
+## Migration
+
+Although `layout` is not available, you can migrate `next/image` to `next/future/image` using a few props. The following is a comparison of the two components:
+
+<table>
+<thead>
+  <tr>
+    <th>next/image</th>
+    <th>next/future/image</th>
+  </tr>
+</thead>
+<tbody>
+
+<tr>
+<td>
+
+```jsx
+import Image from 'next/image'
```

Comment:
This looks a bit strange when live.
<img width="877" alt="CleanShot 2022-08-02 at 19 45 07@2x" src="https://user-images.githubusercontent.com/9113740/182500461-7d1aa90d-f368-4808-9b77-4f93da7bdb47.png">

Thoughts on just doing section headers to for each and just including the after code snippet with a comment in there? 

```jsx
// Replaces default usage of `next/image`
import Image from 'next/future/image'
import img from '../img.png'

const css = { maxWidth: '100%', height: 'auto' }
export default function Page() {
  return <Image src={img} style={css} />
}
```

Reply:
<img width="417" alt="CleanShot 2022-08-03 at 10 54 38@2x" src="https://user-images.githubusercontent.com/9113740/182653975-3a34ea92-60ad-40c1-8e97-aab97aa3ac16.png">

Reply:
> Thoughts on just doing section headers to for each and just including the after code snippet with a comment in there?

Sounds good to me, would be curious if tables would render better if we hid the sidebar 🤔

Reply:
@leerob @ijjk PR with a quick hack: #39309